### PR TITLE
New X-Wing Alliance (videogame) Cheat Sheet

### PR DESCRIPTION
**Pull request by cinlloc**
**What does your Instant Answer do?**
Displays keyboard shortcuts of X-Wing Alliance videogame.

**What is the data source for your Instant Answer? (Provide a link if possible)**
No dynamic datasource (original scanned reference card on Darksaber's X-Wing Station).

**Why did you choose this data source?**
Contains all keyboard shortcuts. 

**Are there any other alternative (better) data sources?**
No. 

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Are you having any problems? Do you need our help with anything?**
No, thanks.

**Where did you hear about DuckDuckHack? (For first time contributors)**
From DuckDuckGo search page!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
xwa

----

IA Page: https://duck.co/ia/view/xwing_alliance_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@cinlloc thanks for your contribution! You've reminded by about the amazing time I spent playing this game with an utterly terrible black-and-green joystick :smile: 

Can you do me a favour though and please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and edit your initial comment? Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

File: share/goodie/cheat_sheets/json/xwing-alliance.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    "description": "Keyboard commands for Starwars X-Wing Alliance simulation game",
+    "metadata": {
+        "sourceName": "Darksaber's X-Wing Station",
+        "sourceUrl": "http://www.darksaber.xwaupgrade.com/archive/files/X-Wing_Alliance_Reference_Card_Printable.pdf"
+    },
+    "aliases": [
+        "xwing",
+        "x wing",
+        "x-wing",
+        "xwing alliance",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/xwing-alliance.json]: This alias should be handled already based on the filename. We take the filename, split it on `-` and create the base trigger from that. So in this case `xwing-alliance` becomes "xwing alliance".

File: share/goodie/cheat_sheets/json/xwing-alliance.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{
+    "id": "xwing_alliance_cheat_sheet",
+    "name": "X-Wing Alliance Shortcuts",
+    "description": "Keyboard commands for Starwars X-Wing Alliance simulation game",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/xwing-alliance.json]: nit: `Star Wars` (casing, and space)

**Comment by moollaza** [comment]
@cinlloc I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

Thanks a lot for making an awesome PR and sticking to the syntax. Once fixed this should be good for merge :+1:

**Comment by cinlloc** [comment]
OK. It's fixed. I'm not sure about aliases: I kept only `xwing alliance` alias, but shouldn't it be nice to keep some of the others to maximize potential matching?

**Comment by moollaza** [comment]
>I kept only xwing alliance alias, but shouldn't it be nice to keep some of the others to maximize potential matching?

Whoops, I only meant you should remove the `xwing alliance` alias, as it was already handled by the filename, so it didn't need to be repeated. The others were fine, though I'm not sure someone searching "xwing cheat sheet" or "xwing reference" really implies they're looking for the X-Wing Alliance game?

**Comment by moollaza** [comment]
@cinlloc This LGTM :+1: I'll go ahead and merged and you can submit another PR with more alises if you'd like. For example, we may also want to add aliases like "star wars x-wing alliance", so "star wars x-wing alliance help" shows this cheat sheet.

**Comment by moollaza** [comment]
@cinlloc I merged this in, and added a few aliases in #1567

**Comment by moollaza** [comment]
BTW, @cinlloc  Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by cinlloc** [comment]
You're welcome. Thanks for your help!

**Comment by moollaza** [comment]
@cinlloc congrats! It's live: https://duckduckgo.com/?q=xwing+alliance+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**